datanode/gateway/graphql/schema.graphql

```diff
@@ -542,6 +542,42 @@ type Query {
     withdrawalId: ID!
   ): Erc20WithdrawalApproval
 
+  "get the signature bundle to add a particular validator to the signer list of the multisig contract"
+  erc20MultiSigSignerAddedBundles(
+    "The node ID of the validator of which a signature bundle is required"
+    nodeId: ID!
+    "The ethereum address of the submitter"
+    submitter: String
+    "The epoch that generated the bundle i.e the epoch in which the node was promoted to tendermint validator"
+    epochSeq: String
```

Comment:
question (non-blocking): how does one get this number out of the schema from the schema as it stands?

Reply:
I'm not sure what you mean by "get it out of the schema" :p if you mean what-will-this-string-look-like it'll be "165" or something. If you mean where-is-it-in-the-response:
```
type ERC20MultiSigSignerAddedBundle {
  "The ethereum address of the signer to be added"
  newSigner: String!
  "The ethereum address of the submitter"
  submitter: String!
  "The nonce used in the signing operation"
  nonce: String!
  "Unix-nano timestamp for when the validator was added"
  timestamp: String!
  "The bundle of signatures from current validators to sign in the new signer"
  signatures: String!
  "The epoch in which the validator was added"
  epochSeq: String! <- it is here
}
```

Reply:
So say i want to query this, I need to pass in epochSeq, nodeId ect.  I can get nodeId from the nodes endpoint, which is cool.. Where do I know what the epochSeq from? There is nothing I see in the node type that tells me this currently or am I missing something?

Reply:
I get you, and thats an interesting point. I suppose you want to ask "which epoch did my node get promoted so I can ask for that bundle" and my answer would be currently you can't (although the data is in the postgres data-base so we could expose it somewhere)

In reality it will always be the bundle with the highest epoch, so you could get them all (and there won't ever be many) and take the highest, but I understand that's not a great workflow!